data/meta/tokens/1/0x8c524635d52bd7b1Bd55E062303177a7d916C046.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdFXS-FXS Pool",
+  "symbol": "sdFXSFXS-f",
+  "description": "This token represents a Curve factory liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains sdFXS and FXS.",
```

Comment:
This pool contains FXS and sdFXS, a tokenized veFXS from StakeDAO.

Reply:
fixed